playbooks/openstack-deploy_network.yml

```diff
@@ -0,0 +1,30 @@
+---
+- hosts: localhost
+  gather_facts: no
+  connection: local
+  roles:
```

Comment:
you don't need ansible-config_template here.

Reply:
Done